core/frontend/src/tile/LayerTileTreeReference.ts

```diff
@@ -0,0 +1,157 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+/** @packageDocumentation
+ * @module Tiles
+ */
+
+import { BaseLayerSettings, ColorDef, MapLayerSettings } from "@itwin/core-common";
+import { SceneContext } from "../ViewContext";
+import { createMapLayerTreeReference, ImageryMapLayerTreeReference, ImageryMapTileTree, LayerTileTree, MapLayerTileTreeReference, ModelMapLayerTileTreeReference, TileTreeLoadStatus, TileTreeReference } from "./internal";
+import { IModelConnection } from "../IModelConnection";
+
+export abstract class LayerTileTreeReference extends TileTreeReference {
```

Comment:
Same comment as for LayerTileTree.

Reply:
The PR has been refactored to not expand the tree hierarchy, so this comment is no longer relevant.